__init__.py

```diff
@@ -129,6 +129,29 @@ def currency_filter(num, grouping=True):
         return locale.currency(num, grouping=grouping)
 
 
+def _unique(values):
+    ret = []
+    for value in values:
+        if value not in ret:
+            ret.append(value)
+    return ret
+
+
+def _fmt_chart_tick(value):
+    return parse_date(value).strftime('%m/%y')
+
+
+@app.template_filter('fmt_chart_ticks')
```

Comment:
Agreed. I think it would be best to pick a convention and stick to it (most of the time, anyway). Speaking of which, especially as the number of filters grows, it would be nice to keep them in their own module, which would make it less necessary to prefix function names with "filter_". It's a little annoying to do, since the filters have to know about the app, but still manageable.